aggregator/builder.go

```diff
@@ -40,6 +41,9 @@ func ProcessAdverts(c *config.Sreality) error {
 		}
 		l = append(l, ConvertSrealityToModel(a, *d))
 	}
+
+	fmt.Println("processed adverts:", len(l))
```

Comment:
What the point to add here this code? Is it related to migrations?

Reply:
it's not related. just adding some info to the app output